make/harbor.cfg

```diff
@@ -133,14 +133,21 @@ clair_db_username = postgres
 
 #Clair default database
 clair_db = postgres
-
-
 ################### end of HA section #####################
+
 #************************END INITIAL PROPERTIES************************
+
 #The following attributes only need to be set when auth mode is uaa_auth
 uaa_endpoint = uaa.mydomain.org
 uaa_clientid = id
 uaa_clientsecret = secret
 uaa_verify_cert = true
 uaa_ca_cert = /path/to/ca.pem
-#############
+
+
+### Docker Registry setting ###
+#registry_storage_provider can be: filesystem, s3, gcs, azure, etc.
+registry_storage_provider = filesystem
+#registry_storage_provider_cfg is a comma separated "key: value" pairs, e.g. "key1: value, key2: value2".
+#Refer to https://docs.docker.com/registry/configuration/#storage for all available configuration.
+registry_storage_provider_cfg = rootdirectory: /storage
```

Comment:
This is tricky, because the `/storage` is a path inside the container, which we mounted to `/data/registry`.  If we don't want to handle it we need to hard code it.

I think the form `k1: v1, k2: v2` is OK but for `filesystem` storage provider, let's just hard code it to `/storage`

Reply:
If the user selects 'filesystem' as the registry storage in Harbor Tile, then we do not provide  text box for the user to specify rootdirectory. So it's hard-coded in Harbor Tile.